gfx/MShockXotto+/pngs_tiles_32x32/character/overlay/wielded/overlay_wielded_scimitar.json

```diff
@@ -0,0 +1 @@
+{"id": "overlay_wielded_scimitar", "fg": ["overlay_wielded_scimitar"], "rotates": false, "bg": []}
```

Comment:
You don't need json if the id and the name of the sprite are the same and you're not using the sprite for several object. So basically if `id` and `fg` are the same you don't need json